CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## 1.1.0
+## 1.1.0-nnbd
```

Comment:
Maybe use `null_safety` instead of `nnbd`. 

Insert "There is no NNBD, I'm Null Safety now" meme.

Reply:
I had originally tried null_safety but underscores aren't allowed, nullsafety one word is also probably fine though. cc @natebosch @kevmoo  wdyt?

Nothing is published yet so I can still change the other packages, we should try to be consistent tho.

Reply:
I have a slight preference for `-nullsafety`

Reply:
talked with nate offline and we agree `-nullsafety` is better